scripts/cat_relations/relationship.py

```diff
@@ -443,272 +454,295 @@ def get_interaction_type(self, positive: bool) -> str:
-                )
-            )
-            if len(in_tags) > 0:
+        for interact in possible_interactions:
+            if not event_for_location(interact.biome):
                 continue
 
-            in_tags = list(
-                filter(
-                    lambda interact_season: interact_season not in _season,
-                    interact.season,
-                )
-            )
-            if len(in_tags) > 0:
+            if not event_for_season(interact.season):
                 continue
 
             if intensity is not None and interact.intensity != intensity:
                 continue
 
             cats_fulfill_conditions = cats_fulfill_single_interaction_constraints(
-                self.cat_from, self.cat_to, interact, game_mode
+                self.cat_from, self.cat_to, interact
             )
             if not cats_fulfill_conditions:
                 continue
 
-            relationship_fulfill_conditions = rel_fulfill_rel_constraints(
-                self, interact.relationship_constraint, interact.id
-            )
-            if not relationship_fulfill_conditions:
-                continue
-
             filtered.append(interact)
 
         return filtered
 
-    # ---------------------------------------------------------------------------- #
-    #                            complex value addition                            #
-    # ---------------------------------------------------------------------------- #
-
-    # How increasing/decreasing one state influences another directly
-    # (an increase of one state doesn't trigger a chain reaction)
-
-    # increase romantic_love -> decreases: dislike | increases: like, comfortable
-    # decrease romantic_love -> decreases: comfortable | increases: -
-
-    # increase like -> decreases: dislike | increases: comfortable
-    # decrease like -> increases: dislike | decreases: comfortable
-
-    # increase dislike -> decreases: romantic_love, like | increases: -
-    # decrease dislike -> increases: like, comfortable | decreases: -
-
-    # increase admiration -> decreases: - | increases: trust
-    # decrease admiration -> increases: dislike | decreases: trust
-
-    # increase comfortable -> decreases: jealousy, dislike | increases: trust, like
-    # decrease comfortable -> increases: jealousy, dislike | decreases: trust, like
-
-    # increase jealousy -> decreases: - | increases: dislike
-    # decrease jealousy -> increases: comfortable | decreases: -
+    def get_amount_of_value(self, value_enum: RelValue) -> Optional[int]:
+        if value_enum == RelValue.ROMANCE:
+            return self.romance
+        elif value_enum == RelValue.LIKE:
+            return self.like
+        elif value_enum == RelValue.RESPECT:
+            return self.respect
+        elif value_enum == RelValue.COMFORT:
+            return self.comfort
+        elif value_enum == RelValue.TRUST:
+            return self.trust
+        else:
+            return None
```

Comment:
oh- oh god. why.

honestly this is just making a case for why relationships should actually be a dict or something bcs wowzers- anyway maybe this?
```suggestion
    def get_amount_of_value(self, value_enum: RelValue) -> Optional[int]:
        return getattr(self, value_enum) if hasattr(self, value_enum) else None
```